test/health-check.test.js

```diff
@@ -1,6 +1,7 @@
 'use strict'
 
 process.env.CACHE_BLOCKS_INFO = 'true'
+process.env.CACHE_BLOCKS_DATA = 'true'
```

Comment:
The actual env var name used is `CACHE_BLOCK_DATA` (no `S`) so this won't have any effect atm.